handler/setup.go

```diff
@@ -48,8 +48,13 @@ func HandleSetup(config *config.Config) http.HandlerFunc {
 			return
 		}
 		id := s.ID
+		if config.Runner.RunOnHostOnly {
+			mountDockerSocket := false
+			s.MountDockerSocket = &mountDockerSocket
+		}
 
 		updateVolumes(s)
+		s.Envs["WORKSPACE"] = getWorkSpacePath(s)
```

Comment:
Let's set all these from Java and avoid adding custom code to the runner.
Currently we are in the middle of an effort to consolidate a lot of these runners to a single repo.
The goal is for the docker runner to not have any custom logic and just get task requests and execute them accordingly.